components/CreateCollective.js

```diff
@@ -78,20 +79,10 @@ class CreateCollective extends React.Component {
   }
 
   async createCollective(CollectiveInputType) {
-    const websiteRegex = /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/;
-
-    if (!CollectiveInputType.name) {
-      this.error('Please enter a name');
-      return;
-    } else if (!CollectiveInputType.description) {
-      this.error('Please enter a description');
-      return;
-    } else if (!CollectiveInputType.website) {
-      this.error('Please enter a website url');
-      return;
-    } else if (CollectiveInputType.description.length > 255) {
+    if (CollectiveInputType.description.length > 255) {
       this.error('The description should be of less than 255 characters');
-    } else if (!websiteRegex.test(CollectiveInputType.website)) {
+      return;
+    } else if (!isURL(CollectiveInputType.website)) {
```

Comment:
From my understanding `isURL('') === false` (see https://codesandbox.io/s/icy-moon-f81we) so the form will be considered as invalid if we don't set a URL.